kustomize/crd/bases/kubecfg.dev_appinstances.yaml

```diff
@@ -53,6 +53,33 @@ spec:
           status:
             nullable: true
             properties:
+              conditions:
+                items:
+                  properties:
+                    lastTransitionTime:
+                      description: Time is a wrapper around time.Time which supports correct marshaling to YAML and JSON.  Wrappers are provided for many of the factory methods that the time package offers.
```

Comment:
I think this is auto-generated from the Go doc? IIRC they wrap `metav1.Time` and it spits this out.

No worries if so, just wondering whether it might be a bit confusing to see the `time.Time` reference in there for a Rust operator.

Reply:
yes this comes from types that are generated from the openapi specs, which in turn are generated from Go code living in the k8s repo